action.yml

```diff
@@ -44,10 +44,12 @@ outputs:
     description: 'Generated Docker tags'
   labels:
     description: 'Generated Docker labels'
-  bake-file:
-    description: 'Bake definiton file'
```

Comment:
I was looking at keeping the `bake-file` output and add a `deprecationMessage` property to it but it's not supported atm. Opened https://github.com/actions/runner/pull/2957